packages/api-gateway-service/.env.example

```diff
@@ -21,4 +21,5 @@ COOKIE_PARSER=
 
 ## Dependent Microservices
 ## User Group
-# USER_GROUP="https://localhost:8080/graphql"
+# USER_SERVICE_SERVICE_HOST="https://localhost:8080/graphql"
+# HOME_SERVICE_SERVICE_HOST="https://localhost:8081/graphql"
```

Comment:
Remove the protocol and `/graphql`. The openshift env variables will just have the hostname.

```suggestion
# USER_SERVICE_SERVICE_HOST="localhost:8080"
# HOME_SERVICE_SERVICE_HOST="localhost:8081"
```

Reply:
Also, add the examples for other microservices, such as Feedback and Notifications.

Reply:
Added other examples